app/api/helpers/permission_manager.py

```diff
@@ -106,7 +106,7 @@ def is_user_itself(view, view_args, view_kwargs, *args, **kwargs):
     Otherwise the user can only access his/her resource.
     """
     user = current_identity
-    if not user.is_admin and not user.is_super_admin and user.id != kwargs['user_id']:
+    if not user.is_admin and not user.is_super_admin and user.id != kwargs['id']:
```

Comment:
1. Please read my comment on the issue, it has already been solved in other PR.
2. This won't work. This will cause other Internal Server errors. For this to work, you will have to modify each and every kwarg passed to this method and convert every instance of ```user_id``` to ```id```.